src/app/products/manage-delinquency-buckets/delinquency-bucket/edit-bucket/edit-bucket.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'mifosx-edit-bucket',
+  templateUrl: './edit-bucket.component.html',
+  styleUrls: ['./edit-bucket.component.scss']
+})
+export class EditBucketComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
```

Comment:
@josehernandezfintecheandomx Is this complete?

Reply:
Done! Completed